web/src/main/java/nl/tehdreamteam/se42/web/command/impl/ExitApplicationCommand.java

```diff
@@ -0,0 +1,37 @@
+package nl.tehdreamteam.se42.web.command.impl;
+
+import nl.tehdreamteam.se42.web.ServiceContainer;
+import nl.tehdreamteam.se42.web.command.Command;
+import nl.tehdreamteam.se42.web.command.CommandHandler;
+
+public class ExitApplicationCommand extends Command {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document this public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)